content/concepts/server-api.md

```diff
@@ -87,41 +75,26 @@ $ curl \
 |:--------:|:---------------------------:|:------------------:|
 | `GET`    | `/v1/ready`                 | `application/json` |
 
-Return the KES server readiness.
+The KES server readiness probe. Returns `200 OK` if KES is ready.
```

Comment:
This is just a writing style thing of using one sentence per line.
It's a technique called "ventilated prose".

```suggestion
The KES server readiness probe. 
Returns `200 OK` if KES is ready.
```